pytorch_lightning/trainer/connectors/checkpoint_connector.py

```diff
@@ -19,21 +19,29 @@
 from abc import ABC
```

Comment:
waiting on #3579 to be merged

Reply:
there is no need, #3579 is independent and this one-time event does not solve anything...